src/test/java/CartTests.java

```diff
@@ -0,0 +1,66 @@
+import hw_at_18_rozetka_page_objects.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CartTests extends BaseTest {
+
+    @Test
+    public void verifyEmptyCart() {
+
+        final String EMPTY_CART_TEXT = "Кошик порожній";
+
+        HomePage homePage = new HomePage(getDriver());
+        homePage.clickCartIcon();
+
+        CartPage cartPage = new CartPage(getDriver());
+
+        Assert.assertTrue(cartPage.getPageTitle().contains(EMPTY_CART_TEXT) && cartPage.imageIsPresent());
+
+    }
+
+    @Test
+    public void verifyProductToCar() {
+        String wordToSearch = "iPhone";
+        int navigateToProduct = 4;
+
+        HomePage homePage = new HomePage(getDriver());
+
+        homePage.typeSearchInput(wordToSearch);
+        homePage.clickSearchButton();
+
+        SearchResultPage searchPage = new SearchResultPage(getDriver());
+
+        String productName = searchPage.getProductName(navigateToProduct);
+        searchPage.addToCart(navigateToProduct);
+        searchPage.clickCartIcon();
+
+        CartPage cartPage = new CartPage(getDriver());
+
+        Assert.assertEquals(productName, cartPage.getProductTitle());
```

Comment:
This test is always failed

Reply:
Default Suite
Total tests run: 1, Passes: 1, Failures: 0, Skips: 0